src/coreclr/src/System.Private.CoreLib/src/System/Globalization/GlobalizationMode.CoreCLR.cs

```diff
@@ -0,0 +1,43 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Diagnostics.CodeAnalysis;
+
+namespace System.Globalization
+{
+    internal static partial class GlobalizationMode
+    {
+        // GetSwitchValue calls CLRConfig first to detect if the switch is defined in the config file.
```

Comment:
This comment is outdated. The method does not check anything that one would call CLRConfig.